demos/canary/netlify.toml

```diff
@@ -3,13 +3,12 @@ command = "next build"
 publish = ".next"
 ignore = "if [ $CACHED_COMMIT_REF == $COMMIT_REF ]; then (exit 1); else git diff --quiet $CACHED_COMMIT_REF $COMMIT_REF ../..; fi;"
 
-[build.environment]
-TERM = "xterm"
-NODE_VERSION = "17"
-
 [[plugins]]
-package = "../plugin-wrapper/"
+package = "@netlify/plugin-nextjs"
```

Comment:
Any reason we're not using the plugin wrapper anymore only in the Canary site?

Reply:
We are (see below in the diff). The reason we have the plugin included by name as well is to prevent build auto-installing the release version (like you had happen). The wrapper now doesn;t actually export anything, it just ensures the monorepo is built. `"@netlify/plugin-nextjs"` is resolved to the one in the monorepo.